python/helpers/jailbreak_utils.py

```
@@ -0,0 +1,633 @@
+            except (subprocess.TimeoutExpired, FileNotFoundError):
+                tools_status[tool] = "❌ Not found"
+
+        # Check for platform-specific tools
+        if self.system == "darwin":  # macOS
+            try:
+                result = subprocess.run(["brew", "--version"],
+                                      capture_output=True,
+                                      text=True,
+                                      timeout=5)
+                tools_status["homebrew"] = "✅ Available"
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)